examples/trials/mnist-nas/config_ppo.yml

```diff
@@ -0,0 +1,20 @@
+authorName: default
```

Comment:
if you think  it's  a general tuner, could provide more benchmark result?

Reply:
sure.